miner/payload_building.go

```diff
@@ -234,7 +234,7 @@ func (w *worker) buildPayload(args *BuildPayloadArgs) (*Payload, error) {
 		// Setup the timer for terminating the process if SECONDS_PER_SLOT (12s in
 		// the Mainnet configuration) have passed since the point in time identified
 		// by the timestamp parameter.
-		endTimer := time.NewTimer(time.Second * 12)
+		endTimer := time.NewTimer(time.Second * 2)
```

Comment:
This is different than the comment above, so it would be nice to leave a quick comment about it.